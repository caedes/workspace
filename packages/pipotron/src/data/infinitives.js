export default [
  'étudier',
  'examiner',
  'favoriser',
  'prendre en considération',
  'anticiper',
  'imaginer',
  'uniformiser',
  'remodeler',
  "avoir à l'esprit",
  'se remémorer',
  'gérer',
  'fédérer',
  'comprendre',
  'analyser',
  'appréhender',
  'expérimenter',
  'essayer',
  'caractériser',
  'façonner',
  'revoir',
  'prendre en compte',
  'arrêter de stigmatiser',
  'considérer',
  'réorganiser',
  'inventorier',
];
