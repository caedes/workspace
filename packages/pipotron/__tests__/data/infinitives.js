import infinitives from '../../src/data/infinitives';

describe('infinitives', () => {
  it('returns the infinitives', () => {
    expect(infinitives).toMatchSnapshot();
  });
});
