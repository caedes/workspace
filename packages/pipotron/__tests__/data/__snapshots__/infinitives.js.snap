// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`infinitives returns the infinitives 1`] = `
Array [
  "étudier",
  "examiner",
  "favoriser",
  "prendre en considération",
  "anticiper",
  "imaginer",
  "uniformiser",
  "remodeler",
  "avoir à l'esprit",
  "se remémorer",
  "gérer",
  "fédérer",
  "comprendre",
  "analyser",
  "appréhender",
  "expérimenter",
  "essayer",
  "caractériser",
  "façonner",
  "revoir",
  "prendre en compte",
  "arrêter de stigmatiser",
  "considérer",
  "réorganiser",
  "inventorier",
]
`;
